internal/pkg/manifest/validate_env_test.go

```diff
@@ -732,41 +732,57 @@ func TestSubnetsConfiguration_validate(t *testing.T) {
 
 func TestCDNConfiguration_validate(t *testing.T) {
 	testCases := map[string]struct {
-		in                   environmentCDNConfig
+		in                   EnvironmentCDNConfig
 		wantedError          error
 		wantedErrorMsgPrefix string
 	}{
 		"valid if empty": {
-			in: environmentCDNConfig{},
+			in: EnvironmentCDNConfig{},
 		},
 		"valid if bool specified": {
-			in: environmentCDNConfig{
+			in: EnvironmentCDNConfig{
 				Enabled: aws.Bool(false),
 			},
 		},
 		"valid if advanced config configured correctly": {
```

Comment:
nit: now that we have `terminate_tls`, we might want to  rename this unit test to something like "success with cert without tls"